geometry_msgs/Pose goal_pose
---
String result
